### PR TITLE
ticket-6

### PR DESCRIPTION
**Pull request by AbdullahIsmail2**

### CODE REVIEW

**Comment by AbdullahIsmail2** [comment]
add comments for each article

File: Api.js [hunk 1]
```diff
@@ -24,3 +24,11 @@ export const getArticlesById = (id) => {
 	});
 };
 
+export const getCommentsByArticleId = (id) => {
+	return newsApi
+		.get(`/api/articles/${id}/comments`)
+		.then(({ data: { comments } }) => {
+			console.log
+			return comments;
+		});
```
Comment (Harrison365) [inline Api.js:33]: Nice 👍

File: src/CommentCard.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	
+	return (
+		<div className="commment-card" style={{marginBottom: '2.5rem'}}>
+			<div style={{display: "flex", justifyContent: 'space-between', marginBottom: '0.5rem'}}>
+				<p style={{fontWeight: '600'}}>{comment.author}</p>
+				<p>{formattedDate}</p>
+			</div>
+			<p>{comment.body}</p>
+		</div>
+	);
```
Comment (Harrison365) [inline src/CommentCard.jsx:15]: Excellent 👍

File: src/FullArticle.jsx [hunk 3]
```diff
@@ -1,41 +1,61 @@
...
 	}, [article_id]);
 
-	console.log(singleArticle);
+	useEffect(() => {
+		if (Object.keys(singleArticle).length > 0) {
+			getCommentsByArticleId(singleArticle.article_id).then((comments) => {
+				setComments(comments);
+			});
+		}
+	}, [singleArticle]);
```
Comment (Harrison365) [inline src/FullArticle.jsx:29]: Looks good :)

File: src/FullArticle.jsx [hunk 4]
```diff
@@ -1,41 +1,61 @@
...
+					</div>
+					<p className="full-article-body">{singleArticle.body}</p>
+				</div>
+				<div
+					className="full-article-comments-container"
+					style={{ width: "85%", margin: "0 auto", marginTop: "2rem" }}
+				>
+					<h4 style={{'marginBottom' : '2rem'}}>Comments</h4>
+					{comments.map((comment) => (
+						<CommentCard comment={comment} />
```
Comment (Harrison365) [inline src/FullArticle.jsx:54]: Nice 👍

File: src/FullArticle.jsx [hunk 5]
```diff
@@ -1,41 +1,61 @@
...
-				<img src={singleArticle.article_img_url} alt="" />
-			</div>
-			<div className="full-article-text-container">
-				<p className="full-article-topic">{`Articles > ${singleArticle.topic}`}</p>
-				<h4 className="full-article-title">{singleArticle.title}</h4>
-				<p className="full-article-author">{singleArticle.author}</p>
-				<div>
-					<p>{`Votes : ${singleArticle.votes}`}</p>
-					<p>{formattedDate}</p>
+		<>
```
Comment (Harrison365) [inline src/FullArticle.jsx:32]: Please add some isLoading functionality whilst the article and comments load :)

**Pull request merged**